master/embedded.html

```diff
@@ -457,17 +457,19 @@ <h2 id="HTMLElements">HTML elements in SVG subtrees</h2>
 
 <edit:example href='images/embedded/sourceinsvg.svg' name='sourceinsvg' link='no' image='no'/>
 
-<p class="note">
-  Currently, the HTML parser does not recognize these tagnames within an SVG subtree 
-  to be HTML-namespaced elements, although this may change in the future. 
-  Therefore, in order to include these elements within your SVG, 
-  you must either use an XML serialization that recognizes namespace declarations 
-  (such as stand-alone SVG or XHTML),
-  or you must create namespaced elements via the <code>createElementNS</code> DOM API method. 
-  Alternatively, you can use the foreignObject element 
-  to wrap the HTML-namespaced elements; 
-  the HTML parser correctly parses HTML elements within a foreignObject.
-</p>
+<div class="note">
+  <p class="note">
```

Comment:
I should probably remove this unnecessary `class="note"`, as it is already specified by the parent element.